### ISSUE TITLE
Fix ENSO Diags behavior when start and end year are equal

### ISSUE THREAD

**Issue by forsyth2** [E3SM-Project/e3sm_diags#298]
ENSO Diags (world map plots and scatterplots) will fail if start and end year are set to the same year. Investigate why and fix.

**Comment by rljacob** [comment]
That kind of makes sense.  Don't you need several years to plot any meaningful ENSO statistics?

**Comment by chengzhuzhang** [comment]
that's true. but it's not expected behavior  based on our current algorithm.
